templates/rails/files/.platform.app.yaml

```diff
@@ -42,8 +42,17 @@ mounts:
 hooks:
     # The build hook turns what is in source control into the deployable application.
     build: |
-        gem install bundler:2.0.2
-        bundle install --without development test
+        gem install --no-document bundler:2.1.4
+        bundle config set without 'development test'
+        bundle config deployment true
+        bundle config cache_all true
+        bundle config clean true
+        echo "Installing gems"
+        [ -d "$PLATFORM_CACHE_DIR/bundle-$PLATFORM_BRANCH" ] && \
+          rsync -az --delete "$PLATFORM_CACHE_DIR/bundle-$PLATFORM_BRANCH/" vendor/bundle/
+        mkdir -p "$PLATFORM_CACHE_DIR/bundle-$PLATFORM_BRANCH"
+        bundle install
+        rsync -az --delete vendor/bundle/ "$PLATFORM_CACHE_DIR/bundle-$PLATFORM_BRANCH/"
```

Comment:
This block should get a comment, since it's non-obvious and these templates serve as documentation for customers with existing repos.

Reply:
Ok i've amended the commit feel free to change it if its not good